cmd/influxd/config.go

```diff
@@ -40,6 +40,10 @@ type Config struct {
 	Version           string `toml:"-"`
 	InfluxDBVersion   string `toml:"-"`
 
+	Authentication struct {
+		Enabled bool `toml:"enabled"`
+	} `toml:"Authentication"`
```

Comment:
Is toml case sensitive? Everything I've seen so far is lower case, `Authenticated` seems out of place.